openedx/openedx-platform
# SE-1151 Campus.il Inlines all 36 discussion underscore templates (fixed)

Fixes bug introduced by https://github.com/edx/edx-platform/pull/20742 (reverted by https://github.com/edx/edx-platform/pull/21049)

**JIRA tickets**: [OSPR-3739](https://openedx.atlassian.net/browse/OSPR-3739)

**Sandbox URL**: TBD - sandbox is being provisioned.

**Merge deadline**:  None

**Testing instructions**:

See https://github.com/edx/edx-platform/pull/20742 for test instructions.

To verify that [the bug introduced by the previous PR](https://github.com/edx/edx-platform/pull/20742#issuecomment-510120191) is repaired:

1. Submit a response to a discussion forum post.
   Note that the author link is clickable, and not displayed as raw HTML.

**Reviewers**
